### DISCUSSION TITLE
Error fetching/processing component definition for Signature Capture

### DISCUSSION THREAD

**Discussion by scamposrw** [pegasystems/constellation-ui-gallery#54] [General]
I imported the RAP file into my Pega application, and had a Signature Capture in the UI, it worked for some time, but from one day to another it started failing:
![image (3)](https://github.com/pegasystems/constellation-ui-gallery/assets/166550773/5acde548-91c0-4bb0-97a7-49a88955a6ab)
At console level when trying to see the Field in the View it throws the following:
![image (4)](https://github.com/pegasystems/constellation-ui-gallery/assets/166550773/0de43664-eb9e-42d7-9d86-52dcc382c00e)
And in the Pega Logs there is an authorization entry being logged regarding a missing or invalid token and if you check the API call for the Signature Capture Component it says this:
![Screenshot 2024-04-26 093902](https://github.com/pegasystems/constellation-ui-gallery/assets/166550773/52c94167-d0ae-41f0-8d7e-1c0ab98feb8b)

The other components I imported seems to be working fine, the issue is with Signature Capture.
Any ideas on the issue ?

**Comment by ricmars** [comment]
check if the component is present - in dev studio, in app explorer - search for Rule-UI-component class and open the instances. If the component is not there, you will need to publish it - if it is there, open the rule and resave - it should force the synchronization to push the code on the C11n app static server

**Reply by scamposrw** [reply]
Yes, it is in the instances of Rule-UI-Component class.
Tried re-importing and revalidating the instance.

**Comment by ricmars** [comment]
issue is not related to this code - this is a sync issue of the component with the c11n server - closing this issue

**Reply by scamposrw** [reply]
In this case is there something we can do in our side? or is something on your side?
Thanks in advance.
